src/main/java/com/CoachPlan/dto/CoachDTO.java

```diff
@@ -3,13 +3,26 @@
 public class CoachDTO {
 	
 	private String loginID;
+	private String fullName;
+	private String Sport;
 	//TODO what other fields should a coach DTO have? Should we have a password field?  
 	
+	public void setName(String name) {
+		this.fullName = name;
+	}
+	
+	public String getName() {
+		return this.fullName;
+	}
 	
-	public void setLoginID(String login) {
```

Comment:
It looks like the setter method was removed, and the getter remains.  Might as well keep the setter.